src/test-util/mockable-imports.ts

```diff
@@ -0,0 +1,14 @@
+export interface Imports {
+  $mock(mocks: object): void;
+  $restore(): void;
+}
+
+/**
+ * Return the `$imports` object for a module that can be used to mock
+ * dependencies in tests.
+ *
+ * @param module - Object containing all the exports of the module to mock
+ */
+export function getImports<T>(module: T): Imports {
+  return (module as any).$imports;
+}
```

Comment:
This is a helper to use in components where we were importing the `$imports` symbol, which is not know by TypeScript.

This is a bit more cumbersome, because it requires a pattern like this:

```ts
import * as SomeComponentExports from '../SomeComponent';

const SomeComponent = SomeComponentExports.default;
const $imports = getImports(SomeComponentExports);
```

Instead of simply this:

```ts
import SomeComponent, { $imports } from '../SomeComponent';
```

I'm looking into this.

Reply:
I just remembered I originally choose this approach because that's what's documented in the babel-plugin docs https://github.com/robertknight/babel-plugin-mockable-imports/tree/master/examples/typescript